sunpy/map/mapbase.py

```diff
@@ -200,6 +200,7 @@ def __init__(self, data, header, plot_settings=None, **kwargs):
         self._default_carrington_longitude = None
         self._default_heliographic_latitude = None
         self._default_heliographic_longitude = None
+        self._wcs = None
```

Comment:
It's a good practice to define all class attributes in the init and not in methods.
```suggestion
        self._meta_hash = None
```